DESCRIPTION

```diff
@@ -41,7 +41,8 @@ Imports:
     r2dii.data,
     rlang,
     tidyr,
-    tidyselect
+    tidyselect,
+    zoo
```

Comment:
I see a new dependency from which we use only one function. It it's simple, I would try to re-implement it (maybe see `zoo:::na.approx.default`). If not so easy, I think it's ok as this package has few if any non-base dependencies, and its maintainers seem very active.

Reply:
it doesn't look super trivial to me and seems to depend on other functions available in `zoo`

Reply:
but i can continue to look! just exploring #239 right now